airflow/www/static/js/dag/details/taskInstance/index.tsx

```diff
@@ -54,18 +52,41 @@ const dagId = getMetaValue('dag_id')!;
 interface Props {
   taskId: Task['id'];
   runId: DagRun['runId'];
+  mapIndex: TaskInstanceType['mapIndex'];
+  onSelect: (selectionProps: SelectionProps) => void;
 }
 
-const TaskInstance = ({ taskId, runId }: Props) => {
-  const [selectedRows, setSelectedRows] = useState<number[]>([]);
+const TaskInstance = ({
+  taskId, runId, mapIndex, onSelect,
+}: Props) => {
+  const isMapIndexDefined = !(mapIndex === undefined);
+  const selectedRows: number[] = isMapIndexDefined ? [mapIndex] : [];
```

Comment:
Do we need selectedRows anymore? We got rid of multiselect.

Reply:
SelectedRow is currently used for the task actions. (Empty <=> all mapped, when non empty only run on those mapIndex).

The name is misleading indeed

Reply:
Let's rename it.

Reply:
done